scripts/testnet/variables.sh

```diff
@@ -14,6 +14,11 @@ export USE_PROXY=1
 # repository (mx-chain-txgen-go).
 export USE_TXGEN=0
 
+# Enable the Elasticsearch data indexing. Will run a Docker image containing an Elasticsearch cluster, on port 9200.
+# It will also change the external.toml files for observers, so they can index data into it
+# docker must be managed as a non-root user: https://docs.docker.com/engine/install/linux-postinstall/
```

Comment:
```suggestion
# It will also change the external.toml files for observers, so they can index data into it.
# Docker must be managed as a non-root user: https://docs.docker.com/engine/install/linux-postinstall/
```